static/js/pages/instances/InstanceList.tsx

```diff
@@ -28,6 +28,19 @@ import StartStopInstanceBtn from "./actions/StartStopInstanceBtn";
 import { useSharedNotify } from "../../context/sharedNotify";
 import OpenTerminalBtn from "./actions/OpenTerminalBtn";
 import OpenVgaBtn from "./actions/OpenVgaBtn";
+import TableColumnsSelect from "components/TableColumnsSelect";
+
+const loadHidden = () => {
+  const saved = localStorage.getItem("InstanceListHidden");
+  if (!saved) {
+    return [];
+  }
+  return JSON.parse(saved) as string[];
+};
+
+const saveHidden = (columns: string[]) => {
+  localStorage.setItem("InstanceListHidden", JSON.stringify(columns));
```

Comment:
Strange naming convention here - I would expect the key to be either camelCased or snake_cased. Not very important though, so feel free to resolve and keep it as is.